app/Repositories/Eloquents/PostRepository.php

```diff
@@ -239,4 +240,30 @@ public function updateVoteTotal($id, $score)
 
         return $temp;
     }
+
+    public function voteBestAnswer($input)
+    {
+        $answer = Answer::where([
+            'post_id' => $input['post_id'],
+            'id' => $input['answer_id']]
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/PostRepository.php</br>**Line**: 248</br>Each line in an array declaration must end in a comma